<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="Functions, tools and utilities for the generation of clinical cancer genome reports with PCGR.
    This R package is an integrated part of the Docker/Conda-based PCGR workflow (https://github.com/sigven/pcgr),
    it should thus not be used as a stand-alone package.">
<title>Personal Cancer Genome ReporteR • pcgr</title>
<script src="deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="deps/bootstrap-5.1.0/bootstrap.min.css" rel="stylesheet">
<script src="deps/bootstrap-5.1.0/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="pkgdown.js"></script><meta property="og:title" content="Personal Cancer Genome ReporteR">
<meta property="og:description" content="Functions, tools and utilities for the generation of clinical cancer genome reports with PCGR.
    This R package is an integrated part of the Docker/Conda-based PCGR workflow (https://github.com/sigven/pcgr),
    it should thus not be used as a stand-alone package.">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    
    <a class="navbar-brand me-2" href="index.html">pcgr</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.10.9</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="active nav-item">
  <a class="nav-link" href="index.html">Intro</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="articles/installation.html">Installation</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="articles/running.html">Running</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="articles/faq.html">FAQ</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="news/index.html">Changelog</a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="nav-link" href="reference/index.html">Reference</a>
</li>
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/sigven/pcgr/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-home">
<div class="row">
  <main id="main" class="col-md-9"><div class="section level1">
<div class="page-header"><h1 id="pcgr">PCGR<a class="anchor" aria-label="anchor" href="#pcgr"></a>
</h1></div>
<p>The Personal Cancer Genome Reporter (PCGR) is a stand-alone software
package for functional annotation and translation of individual cancer
genomes for precision cancer medicine. It interprets both somatic
SNVs/InDels and copy number aberrations. The software extends basic gene
and variant annotations from the <a href="http://www.ensembl.org/info/docs/tools/vep/index.html" class="external-link">Ensembl’s
Variant Effect Predictor (VEP)</a> with oncology-relevant, up-to-date
annotations retrieved flexibly through <a href="https://github.com/brentp/vcfanno" class="external-link">vcfanno</a>. Variants are
classified into <a href="tier_systems.html">tiers of clinical
significance</a>, and interactive HTML output reports permits
exploration of the final results.</p>
<p>Example views from the dashboard HTML output:</p>
<p><img src="img/pcgr_dashboard_views.png"></p>
<p>PCGR has been developed by scientists affiliated with the <a href="http://cancergenomics.no" class="external-link">Norwegian Cancer Genomics
Consortium</a>, at the <a href="http://radium.no" class="external-link">Institute for Cancer
Research/Oslo University Hospital</a>.</p>
<div class="section level2">
<h2 id="example-reports">Example reports<a class="anchor" aria-label="anchor" href="#example-reports"></a>
</h2>
<ul>
<li><a href="http://insilico.hpc.uio.no/pcgr/example_reports/0.9.2/TCGA-FU-A3HZ-01A_TO.pcgr_acmg.grch37.flexdb.html" class="external-link">Cervical
cancer sample (tumor-only)</a></li>
<li><a href="http://insilico.hpc.uio.no/pcgr/example_reports/0.9.2/TCGA-95-7039-01A.pcgr_acmg.grch37.flexdb.html" class="external-link">Lung
cancer sample (tumor-control)</a></li>
<li><a href="http://insilico.hpc.uio.no/pcgr/example_reports/0.9.2/TCGA-EW-A1J5-01A.pcgr_acmg.grch37.flexdb.html" class="external-link">Breast
cancer sample (tumor-control)</a></li>
<li><a href="http://insilico.hpc.uio.no/pcgr/example_reports/0.9.2/TCGA-14-0866-01B.pcgr_acmg.grch37.flexdb.html" class="external-link">Brain
cancer sample (tumor-control)</a></li>
</ul>
<p>(to view the rmarkdown-based reports, simply remove <em>.flexdb.</em>
in the file names for the flexdashboard reports)</p>
</div>
<div class="section level2">
<h2 id="why-use-pcgr">Why use PCGR?<a class="anchor" aria-label="anchor" href="#why-use-pcgr"></a>
</h2>
<p>The great complexity of acquired mutations in individual tumor
genomes poses a severe challenge for clinical interpretation. There is a
general scarcity of tools that can <em>i)</em> systematically
interrogate cancer genomes in the context of diagnostic, prognostic, and
therapeutic biomarkers, <em>ii)</em> prioritize and highlight the most
important findings, and <em>iii)</em> present the results in a format
accessible to clinical experts. PCGR integrates a comprehensive set of
knowledge resources related to tumor biology and therapeutic biomarkers,
both at the gene and variant level. The application generates a tiered
report that will aid the interpretation of individual cancer genomes in
a clinical setting.</p>
<p>If you use PCGR, please cite our publication:</p>
<p>Sigve Nakken, Ghislain Fournous, Daniel Vodák, Lars Birger Aaasheim,
Ola Myklebost, and Eivind Hovig. <strong>Personal Cancer Genome
Reporter: variant interpretation report for precision oncology</strong>
(2017). <em>Bioinformatics</em>. 34(10):1778–1780. <a href="doi:%5B10.1093/bioinformatics/btx817" class="uri">doi:[10.1093/bioinformatics/btx817</a>](<a href="https://doi.org/10.1093/bioinformatics/btx817" class="external-link uri">https://doi.org/10.1093/bioinformatics/btx817</a>)</p>
</div>
<div class="section level2">
<h2 id="docker-based-technology">Docker-based technology<a class="anchor" aria-label="anchor" href="#docker-based-technology"></a>
</h2>
<p>The PCGR workflow is developed using the <a href="https://www.docker.com/what-docker" class="external-link">Docker technology</a>. The
software is thus packaged into isolated containers, in which the
installation of all software libraries/tools and required dependencies
have been taken care of. In addition to the bundled software, in the
form of a Docker image, the workflow only needs to be attached with an
<a href="annotation_resources.html">annotation data bundle for precision
oncology</a>.</p>
<p><img src="img/docker-logo50.png"></p>
</div>
<div class="section level2">
<h2 id="contact">Contact<a class="anchor" aria-label="anchor" href="#contact"></a>
</h2>
<p><a href="mailto:sigven@ifi.uio.no" class="email">sigven@ifi.uio.no</a></p>
</div>
</div>
  </main><aside class="col-md-3"><div class="links">
<h2 data-toc-skip>Links</h2>
<ul class="list-unstyled">
<li><a href="https://github.com/sigven/pcgr/" class="external-link">Browse source code</a></li>
<li><a href="https://github.com/sigven/pcgr/issues" class="external-link">Report a bug</a></li>
</ul>
</div>

<div class="license">
<h2 data-toc-skip>License</h2>
<ul class="list-unstyled">
<li><a href="LICENSE.html">Full license</a></li>
<li><small><a href="https://opensource.org/licenses/mit-license.php" class="external-link">MIT</a> + file <a href="LICENSE-text.html">LICENSE</a></small></li>
</ul>
</div>


<div class="citation">
<h2 data-toc-skip>Citation</h2>
<ul class="list-unstyled">
<li><p><a href="authors.html#citation">Citing PCGR</a></p></li>
</ul>
</div>

<div class="developers">
<h2 data-toc-skip>Developers</h2>
<ul class="list-unstyled">
<li>
<a href="https://github.com/sigven" class="external-link">Sigve Nakken</a> <br><small class="roles"> Author, maintainer </small> <a href="https://orcid.org/0000-0001-8468-2050" target="orcid.widget" aria-label="ORCID" class="external-link"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a> </li>
</ul>
</div>



  </aside>
</div>


    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by <a href="https://github.com/sigven" class="external-link">Sigve
Nakken</a>.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a>
2.0.2.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
