<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="description" content="pcgrr">
<title>Output files â€¢ PCGR</title>
<script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="../deps/bootstrap-5.1.0/bootstrap.min.css" rel="stylesheet">
<script src="../deps/bootstrap-5.1.0/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- bootstrap-toc --><script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Output files">
<meta property="og:description" content="pcgrr">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-light navbar-expand-lg bg-light"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">PCGR</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">0.10.15</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto">
<li class="nav-item">
  <a class="nav-link" href="../index.html">Intro</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="../articles/installation.html">Installation</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="../articles/running.html">Running</a>
</li>
<li class="active nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-articles">Articles</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-articles">
    <a class="dropdown-item" href="../articles/input.html">Input files</a>
    <a class="dropdown-item" href="../articles/output.html">Output files</a>
    <a class="dropdown-item" href="../articles/variant_classification.html">Variant classification</a>
    <a class="dropdown-item" href="../articles/annotation_resources.html">Annotation resources</a>
    <a class="dropdown-item" href="../articles/developers.html">Developer notes</a>
  </div>
</li>
<li class="nav-item">
  <a class="nav-link" href="../articles/faq.html">FAQ</a>
</li>
<li class="nav-item">
  <a class="nav-link" href="../articles/CHANGELOG.html">CHANGELOG</a>
</li>
      </ul>
<form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off">
</form>

      <ul class="navbar-nav">
<li class="nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/sigven/pcgr/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>

    
  </div>
</nav><div class="container template-article">




<div class="row">
  <main id="main"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Output files</h1>
            
      
      <small class="dont-index">Source: <a href="https://github.com/sigven/pcgr/blob/HEAD/../vignettes/output.Rmd" class="external-link"><code>../vignettes/output.Rmd</code></a></small>
      <div class="d-none name"><code>output.Rmd</code></div>
    </div>

    
    
<div class="section level2">
<h2 id="output-files">Output files<a class="anchor" aria-label="anchor" href="#output-files"></a>
</h2>
<p>PCGR generates multiple output files, including interactive HTML
reports and pure variant annotation files (tab-separated values
(TSV)).</p>
<div class="section level3">
<h3 id="html-report---dashboard">HTML report - dashboard<a class="anchor" aria-label="anchor" href="#html-report---dashboard"></a>
</h3>
<p>An interactive and tier-structured HTML report (dashboard-like) that
shows the most relevant findings in the query cancer genome has the
following naming convention:</p>
<p>&lt;<strong>sample_id</strong>&gt;.pcgr_acmg.&lt;<strong>genome_assembly</strong>&gt;.flexdb.html</p>
<ul>
<li>The <strong>sample_id</strong> is provided as input by the user, and
reflects a unique identifier of the tumor-normal sample pair to be
analyzed.</li>
</ul>
</div>
<div class="section level3">
<h3 id="html-report---rmarkdown">HTML report - rmarkdown<a class="anchor" aria-label="anchor" href="#html-report---rmarkdown"></a>
</h3>
<p>An interactive and tier-structured HTML report that shows the most
relevant findings in the query cancer genome has the following naming
convention:</p>
<p>&lt;<strong>sample_id</strong>&gt;.pcgr_acmg.&lt;<strong>genome_assembly</strong>&gt;.html</p>
<ul>
<li>The <strong>sample_id</strong> is provided as input by the user, and
reflects a unique identifier of the tumor-normal sample pair to be
analyzed.</li>
</ul>
<p>The report is structured in seven main sections, described in more
detail below:</p>
<ol style="list-style-type: decimal">
<li>
<strong>Report and query settings</strong>
<ul>
<li>Lists key configurations provided by user</li>
</ul>
</li>
<li>
<strong>Main results</strong>
<ul>
<li>Nine value boxes that highlight the main properties of the tumor
sample:
<ol style="list-style-type: decimal">
<li>Tumor purity - as provided by the user</li>
<li>Tumor ploidy - as provided by the user</li>
<li>Mutational signatures - two most prevalent signatures (other than
aging)</li>
<li>Tier 1 variants (top four)</li>
<li>Tier 2 variants (top four)</li>
<li>Tumor mutational burden (TMB)</li>
<li>Microsatellite instability (MSI) status</li>
<li>Somatic copy number aberrations of clinical significance</li>
<li>Kataegis events</li>
</ol>
</li>
</ul>
</li>
<li>
<strong>Somatic SNVs/InDels</strong>
<ul>
<li>
<em>Tumor mutational burden (TMB)</em>
<ul>
<li>given a coding target region size specified by the user (ideally the
<strong>callable target size</strong>), an estimate of the mutational
burden is provided</li>
<li>The estimated TMB is shown in the context of TMB distributions from
different primary sites in TCGA</li>
</ul>
</li>
<li>
<em>Tier &amp; variant statistics</em>
<ul>
<li>indicate total variant numbers across variant types, coding types
and tiers</li>
</ul>
</li>
<li>
<em>Global distribution - allelic support</em>
<ul>
<li>distribution (histogram) of variant allelic support for somatic
variants (will only be present in the report if specific fields in input
VCF is defined and specified by the user)</li>
</ul>
</li>
<li>
<em>Global variant browser</em>
<ul>
<li>permits exploration of the whole SNV/InDel dataset by filtering
along several dimensions (call confidence, variant sequencing
depth/support, variant consequence etc.)</li>
</ul>
</li>
<li>
<em>Tier tables</em>
<ul>
<li>Variants are organized into five (tier 1-4 + noncoding) interactive
datatables) according to clinical utility</li>
<li>Contents of the tier tables are outlined below</li>
</ul>
</li>
</ul>
</li>
<li>
<strong>Somatic CNAs</strong>
<ul>
<li>
<em>Segments - amplifications and homozygous deletions</em>
<ul>
<li>Based on user-defined/default log-ratio thresholds of gains/losses,
the whole CNA dataset can be navigated further through filters:</li>
<li>cytoband</li>
<li>type of CNA event - <em>focal</em> (less than 25% of chromosome arm
affected) or <em>broad</em>
</li>
<li>log ratio</li>
</ul>
</li>
<li>
<em>Proto-oncogenes subject to copy number amplifications</em>
<ul>
<li>Datatable listing known proto-oncogenes covered by
user-defined/default amplifications and potential targeted
therapies</li>
</ul>
</li>
<li>
<em>Tumor suppressor genes subject to homozygous deletions</em>
<ul>
<li>Datatable listing known tumor suppressor genes covered by
user-defined/default losses and potential targeted therapies</li>
</ul>
</li>
<li><em>Other drug targets subject to copy number
aplifications</em></li>
<li>
<em>Copy number aberrations as biomarkers for prognosis, diagnosis,
and drug response (both of <b>strong</b> significance and
<b>potential</b> significance)</em>
<ul>
<li>Interactive data table where the user can navigate aberrations
acting as biomarkers across therapeutic contexts, tumor subtypes,
evidence levels etc</li>
</ul>
</li>
</ul>
</li>
<li>
<strong>MSI status</strong>
<ul>
<li>Indicates predicted microsatellite stability from the somatic
mutation profile and supporting evidence (details of the underlying MSI
statistical classifier can be found <a href="http://rpubs.com/sigven/msi_classification_v3" class="external-link">here</a>)</li>
<li>The MSI classifier was trained on TCGA exome samples.</li>
</ul>
</li>
<li>
<strong>Mutational signatures</strong>
<ul>
<li>Estimation of relative contribution of <a href="http://cancer.sanger.ac.uk/cosmic/signatures" class="external-link">67 known mutational
signatures</a> in tumor sample (using <a href="https://github.com/raerose01/deconstructSigs" class="external-link">MutationalPatterns</a>
as the underlying framework)</li>
<li>Datatable with signatures and proposed underlying etiologies</li>
</ul>
</li>
<li>
<strong>Kataegis events</strong>
<ul>
<li>Detection of potential kataegis events using the algorithm outlined
in the <a href="https://github.com/MeichunCai/KataegisPortal" class="external-link">Kataegis
Portal</a>
</li>
</ul>
</li>
<li>
<strong>Germline findings</strong>
<ul>
<li>Main findings from CPSR report</li>
</ul>
</li>
<li>
<strong>Clinial trials</strong>
<ul>
<li>Interactive table with annotated, molecularly targeted clinical
trials from <a href="https//clinicaltrials.gov">clinicaltrials.gov</a>
in the relevant tumor type</li>
</ul>
</li>
<li>
<strong>Documentation</strong>
<ul>
<li>Annotation resources - software, databases and tools with version
information</li>
<li>Report content</li>
<li>References - supporting scientific literature (key report
elements)</li>
</ul>
</li>
</ol>
<div class="section level4">
<h4 id="interactive-datatables">Interactive datatables<a class="anchor" aria-label="anchor" href="#interactive-datatables"></a>
</h4>
<p>The interactive datatables contain a number of hyperlinked
annotations similar to those defined for the annotated VCF file,
including the following:</p>
<ul>
<li>SYMBOL - Gene symbol (Entrez/NCBI)</li>
<li>PROTEIN_CHANGE - Amino acid change (VEP)</li>
<li>CANCER_TYPE - Biomarker (tier 1/2): associated cancer type</li>
<li>EVIDENCE_LEVEL - Biomarker (tier 1/2): evidence level
(A,B,C,D,E)</li>
<li>CLINICAL_SIGNIFICANCE - Biomarker (tier 1/2): drug sensitivity,
poor/better outcome etc</li>
<li>EVIDENCE_TYPE - Biomarker (tier 1/2):
predictive/diagnostic/therapeutic</li>
<li>DISEASE_ONTOLOGY_ID - Biomarker (tier 1/2): associated cancer type
(Disease Ontology)</li>
<li>EVIDENCE_DIRECTION - Biomarker (tier 1/2): supports/does not
support</li>
<li>DESCRIPTION - Biomarker (tier 1/2): description</li>
<li>VARIANT_ORIGIN - Biomarker (tier 1/2): variant origin
(germline/somatic)</li>
<li>BIOMARKER_MAPPING - Biomarker (tier 1/2): accuracy of genomic
mapping (exact,codon,exon)</li>
<li>CITATION - Biomarker (tier 1/2): supporting literature</li>
<li>THERAPEUTIC_CONTEXT - Biomarker (tier 1/2): associated drugs</li>
<li>RATING - Biomarker (tier 1/2): trust rating from 1 to 5 (CIVIC)</li>
<li>GENE_NAME - gene name description (Entrez/NCBI)</li>
<li>PROTEIN_DOMAIN - PFAM protein domain</li>
<li>PROTEIN_FEATURE - UniProt feature overlapping variant site</li>
<li>CDS_CHANGE - Coding sequence change</li>
<li>MUTATION_HOTSPOT - Known cancer mutation hotspot</li>
<li>MUTATION_HOTSPOT_CANCERTYPE - Hotspot-associated cancer types</li>
<li>TCGA_FREQUENCY - Frequency of variant in TCGA cohorts</li>
<li>ICGC_PCAWG_OCCURRENCE - Frequency of variant in ICGC-PCAWG
cohorts</li>
<li>DOCM_LITERATURE - Literature links - DoCM</li>
<li>DOCM_DISEASE - Associated diseases - DoCM</li>
<li>OPENTARGETS_RANK - Strength (max across all phenotypes) of phenotype
association according to the Open Targets Platform</li>
<li>OPENTARGETS_ASSOCIATIONS - Phenotype associations with the gene
retrieved from the Open Targets Platform</li>
<li>INTOGEN_DRIVER_MUT - predicted driver mutation - IntOGen</li>
<li>CONSEQUENCE - VEP consequence (primary transcript)</li>
<li>HGVSc - from VEP</li>
<li>HGVSp - from VEP</li>
<li>NCBI_REFSEQ - Transcript accession ID(s) (NCBI RefSeq)</li>
<li>ONCOGENE - Predicted as proto-oncogene from CancerMine/NCG</li>
<li>CANCERGENE_SUPPORT - Links to underlying publications (CancerMine)
that support oncogenic/tumor suppressive role of gene</li>
<li>TUMOR_SUPPRESSOR - Predicted as tumor suppressor gene from
CancerMine/NCG</li>
<li>PREDICTED_EFFECT - Effect predictions from dbNSFP</li>
<li>VEP_ALL_CSQ - All VEP transcript block consequences</li>
<li>DBSNP - dbSNP rsID</li>
<li>COSMIC - Cosmic mutation IDs</li>
<li>CLINVAR - ClinVar variant origin and associated phenotypes</li>
<li>CANCER_ASSOCIATIONS - Gene-associated cancer types from
DisGenet</li>
<li>TARGETED_DRUGS - Targeted drugs from Open Targets Platform
/ChEMBL</li>
<li>KEGG_PATHWAY - Gene-associated pathways from KEGG</li>
<li>CALL_CONFIDENCE - Variant confidence (as set by user in input
VCF)</li>
<li>DP_TUMOR - Variant sequencing depth in tumor (as set by user in
input VCF)</li>
<li>AF_TUMOR - Variant allelic fraction in tumor (as set by user in
input VCF)</li>
<li>DP_CONTROL - Variant sequencing depth in control sample (as set by
user in input VCF)</li>
<li>AF_CONTROL - Variant allelic fraction in control sample (as set by
user in input VCF)</li>
<li>GENOMIC_CHANGE - Variant ID</li>
<li>GENOME_VERSION - Genome assembly</li>
</ul>
<p>Example reports:</p>
<p><a href="https://doi.org/10.5281/zenodo.5045309" class="external-link"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.5045309.svg" alt="DOI"></a></p>
<p>The HTML reports have been tested using the following browsers:</p>
<ul>
<li>Safari (Version 14.1.1 (16611.2.7.1.4))</li>
<li>Mozilla Firefox (83.0)</li>
<li>Google Chrome (Version 90.0.4430.212 (Official Build) (x86_64))</li>
</ul>
</div>
</div>
<div class="section level3">
<h3 id="json">JSON<a class="anchor" aria-label="anchor" href="#json"></a>
</h3>
<p>A compressed JSON file that stores all the essential content of the
report is provided.</p>
<p>This file will easen the process of extracting particular parts of
the report for further analysis or integration with other workflows. The
JSON contains two main objects, <em>metadata</em> and <em>content</em>,
where the former contains information about the settings, data versions,
and the latter contains the various sections of the report.</p>
</div>
<div class="section level3">
<h3 id="snvsindels">SNVs/InDels<a class="anchor" aria-label="anchor" href="#snvsindels"></a>
</h3>
<div class="section level4">
<h4 id="variant-call-format---vcf">1. Variant call format - VCF<a class="anchor" aria-label="anchor" href="#variant-call-format---vcf"></a>
</h4>
<p>A VCF file containing annotated, somatic calls (single nucleotide
variants and insertion/deletions) is generated with the following naming
convention:</p>
<p>&lt;<strong>sample_id</strong>&gt;.pcgr_acmg.&lt;<strong>genome_assembly</strong>&gt;.vcf.gz</p>
<p>Here, the <strong>sample_id</strong> is provided as input by the
user, and reflects a unique identifier of the tumor-normal sample pair
to be analyzed. Following common standards, the annotated VCF file is
compressed with <a href="http://www.htslib.org/doc/tabix.html" class="external-link">bgzip</a>
and indexed with <a href="http://www.htslib.org/doc/tabix.html" class="external-link">tabix</a>. Below follows a
description of all annotations/tags present in the VCF INFO column after
processing with the PCGR annotation pipeline:</p>
<div class="section level5">
<h5 id="vep-consequence-annotations">
<em>VEP consequence annotations</em><a class="anchor" aria-label="anchor" href="#vep-consequence-annotations"></a>
</h5>
<ul>
<li>CSQ - Complete consequence annotations from VEP. Format:
Allele|Consequence|IMPACT|SYMBOL|Gene|Feature_type|Feature|BIOTYPE|EXON|
INTRON|HGVSc|HGVSp|cDNA_position|CDS_position|Protein_position|Amino_acids|
Codons|Existing_variation|ALLELE_NUM|DISTANCE|STRAND|FLAGS|PICK|VARIANT_CLASS|
SYMBOL_SOURCE|HGNC_ID|CANONICAL|MANE_SELECT|MANE_PLUS_CLINICAL|TSL|APPRIS|CCDS|
ENSP|SWISSPROT|TREMBL|UNIPARC|UNIPROT_ISOFORM|
RefSeq|DOMAINS|HGVS_OFFSET|AF|AFR_AF|AMR_AF|EAS_AF|EUR_AF|SAS_AF|gnomAD_AF|
gnomAD_AFR_AF|gnomAD_AMR_AF|gnomAD_ASJ_AF|gnomAD_EAS_AF|gnomAD_FIN_AF|
gnomAD_NFE_AF|gnomAD_OTH_AF|gnomAD_SAS_AF|CLIN_SIG|SOMATIC|PHENO|CHECK_REF|
MOTIF_NAME|MOTIF_POS|HIGH_INF_POS|MOTIF_SCORE_CHANGE|TRANSCRIPTION_FACTORS|NearestExonJB</li>
<li>Consequence - Impact modifier for the consequence type (picked by
VEPâ€™s â€“flag_pick_allele option)</li>
<li>Gene - Ensembl stable ID of affected gene (picked by VEPâ€™s
â€“flag_pick_allele option)</li>
<li>Feature_type - Type of feature. Currently one of Transcript,
RegulatoryFeature, MotifFeature (picked by VEPâ€™s â€“flag_pick_allele
option)</li>
<li>Feature - Ensembl stable ID of feature (picked by VEPâ€™s
â€“flag_pick_allele option)</li>
<li>cDNA_position - Relative position of base pair in cDNA sequence
(picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>CDS_position - Relative position of base pair in coding sequence
(picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>CDS_CHANGE - Coding, transcript-specific sequence annotation (picked
by VEPâ€™s â€“flag_pick_allele option)</li>
<li>AMINO_ACID_START - Protein position indicating absolute start of
amino acid altered (fetched from Protein_position)</li>
<li>AMINO_ACID_END - Protein position indicating absolute end of amino
acid altered (fetched from Protein_position)</li>
<li>Protein_position - Relative position of amino acid in protein
(picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>Amino_acids - Only given if the variant affects the protein-coding
sequence (picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>Codons - The alternative codons with the variant base in upper case
(picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>IMPACT - Impact modifier for the consequence type (picked by VEPâ€™s
â€“flag_pick_allele option)</li>
<li>VARIANT_CLASS - Sequence Ontology variant class (picked by VEPâ€™s
â€“flag_pick_allele option)</li>
<li>SYMBOL - Gene symbol (picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>SYMBOL_ENTREZ - Official gene symbol as provided by NCBIâ€™s Entrez
gene</li>
<li>SYMBOL_SOURCE - The source of the gene symbol (picked by VEPâ€™s
â€“flag_pick_allele option)</li>
<li>STRAND - The DNA strand (1 or -1) on which the transcript/feature
lies (picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>ENSP - The Ensembl protein identifier of the affected transcript
(picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>FLAGS - Transcript quality flags: cds_start_NF: CDS 5â€™, incomplete
cds_end_NF: CDS 3â€™ incomplete (picked by VEPâ€™s â€“flag_pick_allele
option)</li>
<li>SWISSPROT - Best match UniProtKB/Swiss-Prot accession of protein
product (picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>TREMBL - Best match UniProtKB/TrEMBL accession of protein product
(picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>UNIPARC - Best match UniParc accession of protein product (picked by
VEPâ€™s â€“flag_pick_allele option)</li>
<li>HGVSc - The HGVS coding sequence name (picked by VEPâ€™s
â€“flag_pick_allele option)</li>
<li>HGVSp - The HGVS protein sequence name (picked by VEPâ€™s
â€“flag_pick_allele option)</li>
<li>HGVSp_short - The HGVS protein sequence name, short version (picked
by VEPâ€™s â€“flag_pick_allele option)</li>
<li>HGVS_OFFSET - Indicates by how many bases the HGVS notations for
this variant have been shifted (picked by VEPâ€™s â€“flag_pick_allele
option)</li>
<li>NearestExonJB - VEP plugin that finds nearest exon junction for a
coding sequence variant. Format: Ensembl exon identifier+distanceto exon
boundary+boundary type(start/end)+exon length</li>
<li>MOTIF_NAME - The source and identifier of a transcription factor
binding profile aligned at this position (picked by VEPâ€™s
â€“flag_pick_allele option)</li>
<li>MOTIF_POS - The relative position of the variation in the aligned
TFBP (picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>HIGH_INF_POS - A flag indicating if the variant falls in a high
information position of a transcription factor binding profile (TFBP)
(picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>MOTIF_SCORE_CHANGE - The difference in motif score of the reference
and variant sequences for the TFBP (picked by VEPâ€™s â€“flag_pick_allele
option)</li>
<li>CELL_TYPE - List of cell types and classifications for regulatory
feature (picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>CANONICAL - A flag indicating if the transcript is denoted as the
canonical transcript for this gene (picked by VEPâ€™s â€“flag_pick_allele
option)</li>
<li>CCDS - The CCDS identifier for this transcript, where applicable
(picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>INTRON - The intron number (out of total number) (picked by VEPâ€™s
â€“flag_pick_allele option)</li>
<li>EXON - The exon number (out of total number) (picked by VEPâ€™s
â€“flag_pick_allele option)</li>
<li>LAST_EXON - Logical indicator for last exon of transcript (picked by
VEPâ€™s â€“flag_pick_allele option)</li>
<li>LAST_INTRON - Logical indicator for last intron of transcript
(picked by VEPâ€™s â€“flag_pick_allele option)</li>
<li>INTRON_POSITION - Relative position of intron variant to nearest
exon/intron junction (NearestExonJB VEP plugin)</li>
<li>EXON_POSITION - Relative position of exon variant to nearest
intron/exon junction (NearestExonJB VEP plugin)</li>
<li>DISTANCE - Shortest distance from variant to transcript (picked by
VEPâ€™s â€“flag_pick_allele option)</li>
<li>BIOTYPE - Biotype of transcript or regulatory feature (picked by
VEPâ€™s â€“flag_pick_allele option)</li>
<li>TSL - Transcript support level (picked by VEPâ€™s â€“flag_pick_allele
option)&gt;</li>
<li>PUBMED - PubMed ID(s) of publications that cite existing variant -
VEP</li>
<li>PHENO - Indicates if existing variant is associated with a
phenotype, disease or trait - VEP</li>
<li>GENE_PHENO - Indicates if overlapped gene is associated with a
phenotype, disease or trait - VEP</li>
<li>ALLELE_NUM - Allele number from input; 0 is reference, 1 is first
alternate etc - VEP</li>
<li>REFSEQ_MATCH - The RefSeq transcript match status; contains a number
of flags indicating whether this RefSeq transcript matches the
underlying reference sequence and/or an Ensembl transcript (picked by
VEPâ€™s â€“flag_pick_allele option)</li>
<li>PICK - Indicates if this block of consequence data was picked by
VEPâ€™s â€“flag_pick_allele option</li>
<li>VEP_ALL_CONSEQUENCE - All transcript consequences
(Consequence:SYMBOL:Feature_type:Feature:BIOTYPE) - VEP</li>
<li>EXONIC_STATUS - Indicates if variant consequence type is â€˜exonicâ€™ or
â€˜nonexonicâ€™. We here define â€˜exonicâ€™ as any variant with either of the
following consequence:
<ul>
<li>stop_gained / stop_lost</li>
<li>start_lost</li>
<li>frameshift_variant</li>
<li>missense_variant</li>
<li>splice_donor_variant</li>
<li>splice_acceptor_variant</li>
<li>inframe_insertion / inframe_deletion</li>
<li>synonymous_variant</li>
<li>protein_altering</li>
</ul>
</li>
<li>CODING_STATUS - Indicates if primary variant consequence type is
â€˜codingâ€™ or â€˜noncodingâ€™ (wrt. protein-alteration). â€˜codingâ€™ variants are
here defined as those with an â€˜exonicâ€™ status, with the exception of
synonymous variants</li>
</ul>
</div>
<div class="section level5">
<h5 id="gene-information">
<em>Gene information</em><a class="anchor" aria-label="anchor" href="#gene-information"></a>
</h5>
<ul>
<li>ENTREZ_ID - <a href="http://www.ncbi.nlm.nih.gov/gene" class="external-link">Entrez</a>
gene identifier</li>
<li>APPRIS - Principal isoform flags according to the <a href="http://appris.bioinfo.cnio.es/#/downloads" class="external-link">APPRIS principal
isoform database</a>
</li>
<li>MANE_SELECT - Indicating if the transcript is the MANE Select or
MANE Plus Clinical transcript for the gene (picked by VEPâ€™s
â€“flag_pick_allele_gene option)</li>
<li>UNIPROT_ID - <a href="http://www.uniprot.org" class="external-link">UniProt</a>
identifier</li>
<li>UNIPROT_ACC - <a href="http://www.uniprot.org" class="external-link">UniProt</a>
accession(s)</li>
<li>ENSEMBL_GENE_ID - Ensembl gene identifier for VEPâ€™s picked
transcript (<em>ENSGXXXXXXX</em>)</li>
<li>ENSEMBL_TRANSCRIPT_ID - Ensembl transcript identifier for VEPâ€™s
picked transcript (<em>ENSTXXXXXX</em>)</li>
<li>ENSEMBL_PROTEIN_ID - Ensembl corresponding protein identifier for
VEPâ€™s picked transcript</li>
<li>REFSEQ_MRNA - Corresponding RefSeq transcript(s) identifier for
VEPâ€™s picked transcript (<em>NM_XXXXX</em>)</li>
<li>TRANSCRIPT_MANE_SELECT - MANE select transcript identifer: one
high-quality representative transcript per protein-coding gene that is
well-supported by experimental data and represents the biology of the
gene</li>
<li>TRANSCRIPT_MANE_PLUS_CLINICAL - transcripts chosen to supplement
MANE Select when needed for clinical variant reporting</li>
<li>GENCODE_TAG - tag for gencode transcript (basic etc)</li>
<li>GENCODE_TRANSCRIPT_TYPE - type of transcript (protein-coding
etc.)</li>
<li>CORUM_ID - Associated protein complexes (identifiers) from <a href="http://mips.helmholtz-muenchen.de/corum/" class="external-link">CORUM</a>
</li>
<li>TUMOR_SUPPRESSOR - Indicates whether gene is predicted as a tumor
suppressor gene, from Network of Cancer Genes (NCG) &amp; the CancerMine
text-mining resource</li>
<li>TUMOR_SUPPRESSOR_EVIDENCE - Underlying evidence for gene being a
tumor suppressor. Format:
NCG:&lt;TRUE|FALSE&gt;&amp;CancerMine:&lt;LC|MC|HC&gt;:num_citations</li>
<li>ONCOGENE - Indicates whether gene is predicted as an oncogene, from
Network of Cancer Genes (NCG) &amp; the CancerMine text-mining
resource</li>
<li>ONCOGENE_EVIDENCE - Underlying evidence for gene being an oncogene.
Format:
NCG:&lt;TRUE|FALSE&gt;&amp;CancerMine:&lt;LC|MC|HC&gt;:num_citations</li>
<li>INTOGEN_DRIVER - Gene is predicted as a cancer driver in the <a href="https://www.intogen.org/downloads" class="external-link">IntoGen Cancer Drivers
Database</a>
</li>
<li>TCGA_DRIVER - Gene is predicted as a cancer driver in the <a href="https://www.ncbi.nlm.nih.gov/pubmed/29625053" class="external-link">TCGA pan-cancer
analysis of cancer driver genes and mutations</a>
</li>
<li>PROB_EXAC_LOF_INTOLERANT - dbNSFP_gene: the probability of being
loss-of-function intolerant (intolerant of both heterozygous and
homozygous lof variants) based on ExAC r0.3 data</li>
<li>PROB_EXAC_LOF_INTOLERANT_HOM - dbNSFP_gene: the probability of being
intolerant of homozygous, but not heterozygous lof variants based on
ExAC r0.3 data</li>
<li>PROB_EXAC_LOF_TOLERANT_NULL - dbNSFP_gene: the probability of being
tolerant of both heterozygous and homozygous lof variants based on ExAC
r0.3 data</li>
<li>PROB_EXAC_NONTCGA_LOF_INTOLERANT - dbNSFP_gene: the probability of
being loss-of-function intolerant (intolerant of both heterozygous and
homozygous lof variants) based on ExAC r0.3 nonTCGA subset</li>
<li>PROB_EXAC_NONTCGA_LOF_INTOLERANT_HOM - dbNSFP_gene: the probability
of being intolerant of homozygous, but not heterozygous lof variants
based on ExAC r0.3 nonTCGA subset</li>
<li>PROB_EXAC_NONTCGA_LOF_TOLERANT_NULL - dbNSFP_gene: the probability
of being tolerant of both heterozygous and homozygous lof variants based
on ExAC r0.3 nonTCGA subset</li>
<li>PROB_GNOMAD_LOF_INTOLERANT - dbNSFP_gene: the probability of being
loss-of-function intolerant (intolerant of both heterozygous and
homozygous lof variants based on gnomAD 2.1 data</li>
<li>PROB_GNOMAD_LOF_INTOLERANT_HOM - dbNSFP_gene: the probability of
being intolerant of homozygous, but not heterozygous lof variants based
on gnomAD 2.1 data</li>
<li>PROB_GNOMAD_LOF_TOLERANT_NULL - dbNSFP_gene: the probability of
being tolerant of both heterozygous and homozygous lof variants based on
gnomAD 2.1 data</li>
<li>PROB_HAPLOINSUFFICIENCY - dbNSFP_gene: Estimated probability of
haploinsufficiency of the gene (from <a href="http://dx.doi.org/10.1371/journal.pgen.1001154" class="external-link uri">http://dx.doi.org/10.1371/journal.pgen.1001154</a>)</li>
<li>ESSENTIAL_GENE_CRISPR - dbNSFP_gene: Essential (E) or Non-essential
phenotype-changing (N) based on large scale CRISPR experiments. from <a href="http://dx.doi.org/10.1126/science.aac7041" class="external-link uri">http://dx.doi.org/10.1126/science.aac7041</a>
</li>
<li>ESSENTIAL_GENE_CRISPR2 - dbNSFP_gene: Essential (E),
context-Specific essential (S), or Non-essential phenotype-changing (N)
based on large scale CRISPR experiments. from <a href="http://dx.doi.org/10.1016/j.cell.2015.11.015" class="external-link uri">http://dx.doi.org/10.1016/j.cell.2015.11.015</a>
</li>
</ul>
</div>
<div class="section level5">
<h5 id="variant-effect-and-protein-coding-information">
<em>Variant effect and protein-coding information</em><a class="anchor" aria-label="anchor" href="#variant-effect-and-protein-coding-information"></a>
</h5>
<ul>
<li><p>MUTATION_HOTSPOT - mutation hotspot codon in <a href="http://cancerhotspots.org/" class="external-link">cancerhotspots.org</a>. Format:
gene_symbol | codon | q-value</p></li>
<li><p>MUTATION_HOTSPOT_TRANSCRIPT - hotspot-associated transcripts
(Ensembl transcript ID)</p></li>
<li><p>MUTATION_HOTSPOT_CANCERTYPE - hotspot-associated cancer types
(from cancerhotspots.org)</p></li>
<li><p>UNIPROT_FEATURE - Overlapping protein annotations from <a href="http://www.uniprot.org" class="external-link">UniProt KB</a></p></li>
<li><p>PFAM_DOMAIN - Pfam domain identifier (from VEP)</p></li>
<li><p>INTOGEN_DRIVER_MUT - Indicates if existing variant is predicted
as driver mutation from IntoGen Catalog of Driver Mutations</p></li>
<li><p>PUTATIVE_DRIVER_MUTATION - Variant is predicted as driver
mutation in the <a href="https://www.ncbi.nlm.nih.gov/pubmed/29625053" class="external-link">TCGA pan-cancer
analysis of cancer driver genes and mutations</a></p></li>
<li>
<p>EFFECT_PREDICTIONS - All predictions of effect of variant on
protein function and pre-mRNA splicing from <a href="https://sites.google.com/site/jpopgen/dbNSFP" class="external-link">database of
non-synonymous functional predictions - dbNSFP v4.2</a>. Predicted
effects are provided by different sources/algorithms (separated by
â€˜&amp;â€™), T = Tolerated, N = Neutral, D = Damaging:</p>
<ol style="list-style-type: decimal">
<li><a href="https://sift.bii.a-star.edu.sg/" class="external-link">SIFT</a></li>
<li>
<a href="http://www.mutationtaster.org/" class="external-link">MutationTaster</a> (data
release Nov 2015)</li>
<li>
<a href="http://mutationassessor.org/" class="external-link">MutationAssessor</a> (release
3)</li>
<li>
<a href="http://fathmm.biocompute.org.uk" class="external-link">FATHMM</a> (v2.3)</li>
<li>
<a href="http://provean.jcvi.org/index.php" class="external-link">PROVEAN</a> (v1.1 Jan
2015)</li>
<li><a href="http://fathmm.biocompute.org.uk/fathmmMKL.htm" class="external-link">FATHMM_MKL</a></li>
<li><a href="https://www.nature.com/articles/s41588-018-0167-z" class="external-link">PRIMATEAI</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5570203/" class="external-link">DEOGEN2</a></li>
<li>
<a href="https://www.biorxiv.org/content/10.1101/2021.04.09.438706v1" class="external-link">DBNSFP_CONSENSUS_RNN</a>
(Ensembl/consensus prediction, based on deep learning)</li>
<li>
<a href="http://nar.oxfordjournals.org/content/42/22/13534" class="external-link">SPLICE_SITE_EFFECT_ADA</a>
(Ensembl/consensus prediction of splice-altering SNVs, based on adaptive
boosting)</li>
<li>
<a href="http://nar.oxfordjournals.org/content/42/22/13534" class="external-link">SPLICE_SITE_EFFECT_RF</a>
(Ensembl/consensus prediction of splice-altering SNVs, based on random
forest)</li>
<li><a href="http://bejerano.stanford.edu/MCAP" class="external-link">M-CAP</a></li>
<li><a href="http://mutpred.mutdb.org" class="external-link">MutPred</a></li>
<li><a href="http://mendel.stanford.edu/SidowLab/downloads/gerp/" class="external-link">GERP</a></li>
<li><a href="https://doi.org/10.1002/humu.23158" class="external-link">BayesDel</a></li>
<li><a href="https://doi.org/10.1093/nar/gkaa288" class="external-link">LIST-S2</a></li>
<li><a href="https://www.nature.com/articles/s41467-017-00443-5" class="external-link">ALoFT</a></li>
</ol>
</li>
<li><p>DBNSFP_BAYESDEL_ADDAF - predicted effect from BayesDel
(dbNSFP)</p></li>
<li><p>DBNSFP_LIST_S2 - predicted effect from LIST-S2 (dbNSFP)</p></li>
<li><p>DBNSFP_SIFT - predicted effect from SIFT (dbNSFP)</p></li>
<li><p>DBNSFP_PROVEAN - predicted effect from PROVEAN (dbNSFP)</p></li>
<li><p>DBNSFP_MUTATIONTASTER - predicted effect from MUTATIONTASTER
(dbNSFP)</p></li>
<li><p>DBNSFP_MUTATIONASSESSOR - predicted effect from MUTATIONASSESSOR
(dbNSFP)</p></li>
<li><p>DBNSFP_M_CAP - predicted effect from M-CAP (dbNSFP)</p></li>
<li><p>DBNSFP_ALOFTPRED - predicted effect from ALoFT (dbNSFP)</p></li>
<li><p>DBNSFP_MUTPRED - score from MUTPRED (dbNSFP)</p></li>
<li><p>DBNSFP_FATHMM - predicted effect from FATHMM (dbNSFP)</p></li>
<li><p>DBNSFP_PRIMATEAI - predicted effect from PRIMATEAI
(dbNSFP)</p></li>
<li><p>DBNSFP_DEOGEN2 - predicted effect from DEOGEN2 (dbNSFP)</p></li>
<li><p>DBNSFP_GERP - evolutionary constraint measure from GERP
(dbNSFP)</p></li>
<li><p>DBNSFP_FATHMM_MKL - predicted effect from FATHMM-mkl
(dbNSFP)</p></li>
<li><p>DBNSFP_META_RNN - predicted effect from ensemble prediction (deep
learning - dbNSFP)</p></li>
<li><p>DBNSFP_SPLICE_SITE_RF - predicted effect of splice site
disruption, using random forest (dbscSNV)</p></li>
<li><p>DBNSFP_SPLICE_SITE_ADA - predicted effect of splice site
disruption, using boosting (dbscSNV)</p></li>
</ul>
</div>
<div class="section level5">
<h5 id="variant-frequenciesannotations-in-germlinesomatic-databases">
<em>Variant frequencies/annotations in germline/somatic
databases</em><a class="anchor" aria-label="anchor" href="#variant-frequenciesannotations-in-germlinesomatic-databases"></a>
</h5>
<ul>
<li>AFR_AF_GNOMAD - African/American germline allele frequency (<a href="http://gnomad.broadinstitute.org/" class="external-link">Genome Aggregation Database
release 2</a>)</li>
<li>AMR_AF_GNOMAD - American germline allele frequency (<a href="http://gnomad.broadinstitute.org/" class="external-link">Genome Aggregation Database
release 2</a>)</li>
<li>GLOBAL_AF_GNOMAD - Adjusted global germline allele frequency (<a href="http://gnomad.broadinstitute.org/" class="external-link">Genome Aggregation Database
release 2</a>)</li>
<li>SAS_AF_GNOMAD - South Asian germline allele frequency (<a href="http://gnomad.broadinstitute.org/" class="external-link">Genome Aggregation Database
release 2</a>)</li>
<li>EAS_AF_GNOMAD - East Asian germline allele frequency (<a href="http://gnomad.broadinstitute.org/" class="external-link">Genome Aggregation Database
release 2</a>)</li>
<li>FIN_AF_GNOMAD - Finnish germline allele frequency (<a href="http://gnomad.broadinstitute.org/" class="external-link">Genome Aggregation Database
release 2</a>)</li>
<li>NFE_AF_GNOMAD - Non-Finnish European germline allele frequency (<a href="http://gnomad.broadinstitute.org/" class="external-link">Genome Aggregation Database
release 2</a>)</li>
<li>OTH_AF_GNOMAD - Other germline allele frequency (<a href="http://gnomad.broadinstitute.org/" class="external-link">Genome Aggregation Database
release 2</a>)</li>
<li>ASJ_AF_GNOMAD - Ashkenazi Jewish allele frequency (<a href="http://gnomad.broadinstitute.org/" class="external-link">Genome Aggregation Database
release 2</a>)</li>
<li>AFR_AF_1KG - <a href="http://www.1000genomes.org" class="external-link">1000G Project -
phase 3</a> germline allele frequency for samples from AFR
(African)</li>
<li>AMR_AF_1KG - <a href="http://www.1000genomes.org" class="external-link">1000G Project -
phase 3</a> germline allele frequency for samples from AMR (Ad Mixed
American)</li>
<li>EAS_AF_1KG - <a href="http://www.1000genomes.org" class="external-link">1000G Project -
phase 3</a> germline allele frequency for samples from EAS (East
Asian)</li>
<li>EUR_AF_1KG - <a href="http://www.1000genomes.org" class="external-link">1000G Project -
phase 3</a> germline allele frequency for samples from EUR
(European)</li>
<li>SAS_AF_1KG - <a href="http://www.1000genomes.org" class="external-link">1000G Project -
phase 3</a> germline allele frequency for samples from SAS (South
Asian)</li>
<li>GLOBAL_AF_1KG - <a href="http://www.1000genomes.org" class="external-link">1000G Project -
phase 3</a> germline allele frequency for all 1000G project samples
(global)</li>
<li>DBSNPRSID - <a href="http://www.ncbi.nlm.nih.gov/SNP/" class="external-link">dbSNP</a>
reference ID, as provided by VEP</li>
<li>COSMIC_MUTATION_ID - Mutation identifier in <a href="http://cancer.sanger.ac.uk/cancergenome/projects/cosmic/" class="external-link">Catalog
of somatic mutations in cancer</a> database, as provided by VEP</li>
<li>TCGA_PANCANCER_COUNT - Raw variant count across all TCGA tumor
types</li>
<li>TCGA_FREQUENCY - Frequency of variant across TCGA tumor types.
Format: tumortype| percent affected|affected cases|total cases</li>
<li>ICGC_PCAWG_OCCURRENCE - Mutation occurrence in <a href="http://docs.icgc.org/pcawg/" class="external-link">ICGC-PCAWG</a>. By project:
project_code|affected_donors|tested_donors|frequency)</li>
<li>ICGC_PCAWG_AFFECTED_DONORS - Number of donors with the current
mutation in <a href="http://docs.icgc.org/pcawg/" class="external-link">ICGC-PCAWG</a>
</li>
</ul>
</div>
<div class="section level5">
<h5 id="clinical-associations">
<em>Clinical associations</em><a class="anchor" aria-label="anchor" href="#clinical-associations"></a>
</h5>
<ul>
<li>CLINVAR_MSID - <a href="http://www.ncbi.nlm.nih.gov/clinvar" class="external-link">ClinVar</a> Measure
Set/Variant ID</li>
<li>CLINVAR_ALLELE_ID - <a href="http://www.ncbi.nlm.nih.gov/clinvar" class="external-link">ClinVar</a> allele ID</li>
<li>CLINVAR_PMID - Associated Pubmed IDs for variant in <a href="http://www.ncbi.nlm.nih.gov/clinvar" class="external-link">ClinVar</a> - germline
state-of-origin</li>
<li>CLINVAR_HGVSP - Protein variant expression using HGVS
nomenclature</li>
<li>CLINVAR_PMID_SOMATIC - Associated Pubmed IDs for variant in <a href="http://www.ncbi.nlm.nih.gov/clinvar" class="external-link">ClinVar</a> - somatic
state-of-origin</li>
<li>CLINVAR_CLNSIG - Clinical significance for variant in <a href="http://www.ncbi.nlm.nih.gov/clinvar" class="external-link">ClinVar</a> - germline
state-of-origin</li>
<li>CLINVAR_CLNSIG_SOMATIC - Clinical significance for variant in <a href="http://www.ncbi.nlm.nih.gov/clinvar" class="external-link">ClinVar</a> - somatic
state-of-origin</li>
<li>CLINVAR_MEDGEN_CUI - Associated <a href="https://www.ncbi.nlm.nih.gov/medgen/" class="external-link">MedGen</a> concept
identifiers (<em>CUIs</em>) - germline state-of-origin</li>
<li>CLINVAR_MEDGEN_CUI_SOMATIC - Associated <a href="https://www.ncbi.nlm.nih.gov/medgen/" class="external-link">MedGen</a> concept
identifiers (<em>CUIs</em>) - somatic state-of-origin</li>
<li>CLINVAR_VARIANT_ORIGIN - Origin of variant (somatic, germline, de
novo etc.) for variant in <a href="http://www.ncbi.nlm.nih.gov/clinvar" class="external-link">ClinVar</a>
</li>
<li>CLINVAR_REVIEW_STATUS_STARS - Rating of the <a href="http://www.ncbi.nlm.nih.gov/clinvar" class="external-link">ClinVar</a> variant (0-4
stars) with respect to level of review</li>
<li>DOCM_PMID - Associated Pubmed IDs for variant in <a href="http://docm.genome.wustl.edu" class="external-link">Database of Curated
Mutations</a>
</li>
<li>OPENTARGETS_DISEASE_ASSOCS - Associations between protein targets
and disease based on multiple lines of evidence (mutations,affected
pathways,GWAS, literature etc). Format:
CUI:EFO_ID:IS_DIRECT:OVERALL_SCORE</li>
<li>OPENTARGETS_TRACTABILITY_COMPOUND - Confidence for the existence of
a modulator (small molecule) that interacts with the target to elicit a
desired biological effect</li>
<li>OPENTARGETS_TRACTABILITY_ANTIBODY - Confidence for the existence of
a modulator (antibody) that interacts with the target to elicit a
desired biological effect</li>
</ul>
</div>
<div class="section level5">
<h5 id="other">
<em>Other</em><a class="anchor" aria-label="anchor" href="#other"></a>
</h5>
<ul>
<li>CHEMBL_COMPOUND_ID - antineoplastic drugs targeting the encoded
protein (from <a href="https://www.targetvalidation.org/" class="external-link">Open Targets
Platform</a>, drugs are listed as <a href="https://www.ebi.ac.uk/chembl/" class="external-link">ChEMBL</a> compound
identifiers)</li>
<li>CIVIC_ID, CIVIC_ID_SEGMENT - Variant/segment (exon, codon)
identifiers in the <a href="http://civic.genome.wustl.edu" class="external-link">CIViC
database</a>
</li>
<li>CGI_ID, CGI_ID_SEGMENT - Variant/segment (exon, codon) identifier in
the <a href="https://www.cancergenomeinterpreter.org/biomarkers" class="external-link">Cancer
Genome Interpreter Cancer Biomarkers Database</a>
</li>
</ul>
</div>
</div>
<div class="section level4">
<h4 id="tab-separated-values-tsv">2. Tab-separated values (TSV)<a class="anchor" aria-label="anchor" href="#tab-separated-values-tsv"></a>
</h4>
<div class="section level5">
<h5 id="list-of-all-snvsindels">List of all SNVs/InDels<a class="anchor" aria-label="anchor" href="#list-of-all-snvsindels"></a>
</h5>
<p>We provide a tab-separated values file with most important
annotations for SNVs/InDels. The file has the following naming
convention:</p>
<p>&lt;<strong>sample_id</strong>&gt;.pcgr_acmg.&lt;<strong>genome_assembly</strong>&gt;.snvs_indels.tiers.tsv</p>
<p>The SNVs/InDels are organized into different <strong>tiers</strong>
(as defined above for the HTML report)</p>
<p>The following variables are included in the tiered TSV file:</p>
<pre><code>1. CHROM - Chromosome
2. POS - Position (VCF-based)
3. REF - Reference allele
4. ALT - Alternate allele
5. GENOMIC_CHANGE - Identifier for variant at the genome (VCF) level, e.g. 1:g.152382569A&gt;G
      Format: (&lt;chrom&gt;:g.&lt;position&gt;&lt;ref_allele&gt;&gt;&lt;alt_allele&gt;)
6. GENOME_VERSION - Assembly version, e.g. GRCh37
7. VCF_SAMPLE_ID - Sample identifier
8. VARIANT_CLASS - Variant type, e.g. SNV/insertion/deletion
9. SYMBOL - Gene symbol
10. GENE_NAME - Gene description
11. CCDS - CCDS identifier
12. CANONICAL - indication of canonical transcript
13. ENTREZ_ID - Entrez gene identifier
14. UNIPROT_ID - UniProt protein identifier
15. ENSEMBL_TRANSCRIPT_ID - Ensembl transcript identifier
16. ENSEMBL_GENE_ID - Ensembl gene identifier
17. REFSEQ_MRNA - RefSeq mRNA identifier
18. REFSEQ_PEPTIDE - RefSeq peptide identifier
19. TRANSCRIPT_MANE_SELECT - MANE select transcript identifier
20. ONCOGENE - Gene is predicted/classified as an oncogene (CancerMine/NCG)
21. ONCOGENE_EVIDENCE - Underlying evidence for oncogene prediction/classification
22. TUMOR_SUPPRESSOR - Gene is predicted/classified as tumor suppressor (CancerMine/NCG)
23. TUMOR_SUPPRESSOR_EVIDENCE - Underlying evidence for tumor suppressor prediction/classification
24. CONSEQUENCE - Variant consequence (as defined above for VCF output:
    Consequence)
25. PROTEIN_CHANGE - Protein change (HGVSp without reference accession)
26. PROTEIN_DOMAIN - Protein domain description (Pfam)
27. CODING_STATUS - Coding variant status wrt. protein alteration ('coding' or 'noncoding')
28. EXONIC_STATUS - Exonic variant status ('exonic' or 'nonexonic')
29. CDS_CHANGE - composite VEP-based variable for coding change, format:
    Consequence:Feature:cDNA_position:EXON:HGVSp_short
30. HGVSp
31. HGVSc
32. REGULATORY_ANNOTATION - Regulatory annotations (promoter/enhancer/TF_binding site overlap etc.) from VEP (optional)
33. EFFECT_PREDICTIONS - as defined above for VCF
34. MUTATION_HOTSPOT - mutation hotspot codon in
    cancerhotspots.org. Format: gene_symbol | codon | q-value
35. MUTATION_HOTSPOT_TRANSCRIPT - hotspot-associated transcripts (Ensembl transcript ID)
36. MUTATION_HOTSPOT_CANCERTYPE - hotspot-associated cancer types (from cancerhotspots.org)
37. PUTATIVE_DRIVER_MUTATION - Indicates if variant is predicted as
    driver mutation from TCGA's PanCancer study of cancer driver mutation
38. CHASMPLUS_DRIVER - Driver mutation predicted by CHASMplus algorithm
39. CHASMPLUS_TTYPE - Tumor type for which mutation is predicted as driver by CHASMplus
40. VEP_ALL_CSQ - all VEP transcript block consequences
41. DBSNPRSID - dbSNP reference cluster ID
42. COSMIC_MUTATION_ID - COSMIC mutation ID
43. TCGA_PANCANCER_COUNT - Raw variant count across all TCGA tumor types
44. TCGA_FREQUENCY - Frequency of variant across TCGA tumor types. Format: tumortype|
percent affected|affected cases|total cases
45. ICGC_PCAWG_OCCURRENCE - Mutation occurrence in ICGC-PCAWG by project:
project_code|affected_donors|tested_donors|frequency
46. CHEMBL_COMPOUND_ID - Compounds (as ChEMBL IDs) that are targeted towards the encoded protein (from Open Targets Platform)
47. CHEMBL_COMPOUND_TERMS - Compounds (as drug names) that are targeted towards the encoded protein (from Open Targets Platform)
48. SIMPLEREPEATS_HIT - Variant overlaps UCSC _simpleRepeat_ sequence repeat track
49. WINMASKER_HIT - Variant overlaps UCSC _windowmaskerSdust_ sequence repeat track
50. OPENTARGETS_RANK - OpenTargets association score (between 0 and 1) for gene (maximum across cancer phenotypes)
51. CLINVAR - ClinVar association: variant origin and associated traits
52. CLINVAR_CLNSIG - clinical significance of ClinVar variant
53. GLOBAL_AF_GNOMAD - global germline allele frequency in gnomAD
54. GLOBAL_AF_1KG - 1000G Project - phase 3, germline allele frequency
55. NCER_PERCENTILE - Genome-wide percentile score (10bp bins) from the non-coding Essential Regulation (ncER) algorithm
56. CALL_CONFIDENCE - confidence indicator for somatic variant
57. DP_TUMOR - sequencing depth at variant site (tumor sample)
58. AF_TUMOR - allelic fraction of alternate allele (tumor sample)
59. DP_CONTROL - sequencing depth at variant site (control sample)
60. AF_CONTROL - allelic fraction of alternate allele (control sample)
61. TIER
62. TIER_DESCRIPTION</code></pre>
<p><strong>NOTE</strong>: The user has the possibility to append the TSV
file with data from other tags in the input VCF of interest (i.e.Â using
the <em>custom_tags</em> option in the TOML configuration file)</p>
</div>
<div class="section level5">
<h5 id="mutational-signature-contributions">Mutational signature contributions<a class="anchor" aria-label="anchor" href="#mutational-signature-contributions"></a>
</h5>
<p>We provide a tab-separated values file information about mutational
signatures detected in the tumor sample. The file has the following
naming convention:</p>
<p>&lt;<strong>sample_id</strong>&gt;.pcgr_acmg.&lt;<strong>genome_assembly</strong>&gt;.mutational_signatures.tsv</p>
<p>The format of the TSV file is the following:</p>
<pre><code>1. signature_id - identifier for signature
2. sample_id - sample identifier
3. prop_signature - relative contribution of mutational signature
4. group - keyword for signature aetiology
5. all_reference_signatures - logical indicating if all reference signatures were used for reconstruction/inference
6. tumor_type - tumor type (used for retrieval of reference signatures)
7. reference_collection - collection used for reference signatures
8. reference_signatures - signatures present in reference collection
9. fitting_accuracy - accuracy of mutational signature fitting</code></pre>
</div>
</div>
</div>
<div class="section level3">
<h3 id="copy-number-aberrations">Copy number aberrations<a class="anchor" aria-label="anchor" href="#copy-number-aberrations"></a>
</h3>
<div class="section level4">
<h4 id="tab-separated-values-tsv-1">1. Tab-separated values (TSV)<a class="anchor" aria-label="anchor" href="#tab-separated-values-tsv-1"></a>
</h4>
<p>Copy number segments are intersected with the genomic coordinates of
all transcripts from <a href="https://www.gencodegenes.org/releases/current.html" class="external-link">GENCODEâ€™s
basic gene annotation</a>. In addition, PCGR attaches cancer-relevant
annotations for the affected transcripts. The naming convention of the
compressed TSV file is as follows:</p>
<p>&lt;<strong>sample_id</strong>&gt;.pcgr_acmg.&lt;<strong>genome_assembly</strong>&gt;.cna_segments.tsv.gz</p>
<p><strong>NOTE</strong>: This is file is organized according to the
<em>affected transcripts</em> (i.e.Â one line/record per affected
transcript).</p>
<p>The format of the compressed TSV file is the following:</p>
<pre><code>1. chrom - chromosome
2. segment_start - start of copy number segment
3. segment_end - end of copy number segment
4. segment_id - segment identifier
5. segment_length_Mb - length of segment in Mb
6. event_type - focal or broad (covering more than 25% of chromosome arm)
7. cytoband(s) - cytobands covered by segment
8. LogR - Copy log-ratio
9. sample_id - Sample identifier
10. ensembl_gene_id
11. symbol - gene symbol
12. ensembl_transcript_id
13. transcript_start - chromosomal position of transcript start
14. transcript_end - chromosomal position of transcript end
15. transcript_overlap_percent - percent of transcript length covered by CN segment
16. name - gene name description
17. biotype - type of gene
18. tumor_suppressor - tumor suppressor gene status (CancerMine literature database/Network of Cancer Genes)
19. oncogene - oncogene status (CancerMine literature database/Network of Cancer Genes)
20. intogen_drivers - predicted driver gene status (IntoGen Cancer Drivers Database)
21. chembl_compound_id - anti-cancer drugs that are targeted towards the encoded protein (from Open Targets Platform, drugs are listed as ChEMBL compound identifiers)
22. gencode_tag
23. gencode_release</code></pre>
</div>
</div>
</div>
  </main><aside class="col-md-3"><nav id="toc"><h2>On this page</h2>
    </nav></aside>
</div>



    <footer><div class="pkgdown-footer-left">
  <p></p>
<p>Developed by <a href="https://github.com/sigven" class="external-link">Sigve Nakken</a>,
<a href="https://github.com/pdiakumis" class="external-link">Peter Diakumis</a>.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a>
2.0.2.</p>
</div>

    </footer>
</div>

  

  

  </body>
</html>
